charts/illumidesk/templates/illumidesk-config-map.yaml

```diff
@@ -19,6 +19,7 @@ data:
   LTI13_ENDPOINT: {{ .Values.illumideskSettings.lti13Endpoint | default "https://illumidesk.instructure.com/api/lti/security/jwks"  | quote }}
   LTI13_AUTHORIZE_URL: {{ .Values.illumideskSettings.lti13AuthorizeUrl | default "https://illumidesk.instructure.com/api/lti/authorize_redirect"  | quote }}
   LTI13_TOKEN_URL: {{ .Values.illumideskSettings.lti13TokenUrl | default "https://illumidesk.instructure.com/login/oauth2/token"  | quote }}
+  ACTIVE_USER_LIMIT: {{ .Values.illumideskSettings.active_user_limit | default "250" }}
```

Comment:
Let's set the default `0` since the upstream JupyterHub doesn't set a limit by specifying `0` as [the default](https://jupyterhub.readthedocs.io/en/stable/api/app.html#jupyterhub.app.JupyterHub).

Reply:
updated the default to 0